flink-table/flink-table-api-java/src/main/java/org/apache/flink/table/api/TableEnvironment.java

```diff
@@ -191,9 +191,42 @@ static TableEnvironment create(EnvironmentSettings settings) {
 	 * @return The resulting {@link Table}.
 	 * @see TableEnvironment#useCatalog(String)
 	 * @see TableEnvironment#useDatabase(String)
+	 * @deprecated use {@link #from(String)}
 	 */
+	@Deprecated
 	Table scan(String... tablePath);
 
+	/**
+	 * Reads a registered table and returns the resulting {@link Table}.
+	 *
+	 * <p>A table to scan must be registered in the {@link TableEnvironment}.
+	 *
+	 * <p>See the documentation of {@link TableEnvironment#useDatabase(String)} or
+	 * {@link TableEnvironment#useCatalog(String)} for the rules on the path resolution.
+	 *
+	 * <p>Examples:
+	 *
+	 * <p>Reading a table from default catalog & database.
```

Comment:
We could use this method for describing the general resolution rules with escaping and SQL reserved keywords.  More examples here like "db.Name" and backticks?